src/core/expr/fexpr_minmax.cc

```diff
@@ -52,19 +52,21 @@ class FExpr_MinMax : public FExpr_ReduceUnary {
           return Column(new ConstNa_ColumnImpl(gby.size(), stype));
         case SType::BOOL:
         case SType::INT8:
-          return make<int8_t>(std::move(col), gby, is_grouped);
+          return make<int8_t>(std::move(col), SType::INT8, gby, is_grouped);
         case SType::INT16:
-          return make<int16_t>(std::move(col), gby, is_grouped);
+          return make<int16_t>(std::move(col), SType::INT16, gby, is_grouped);
         case SType::INT32:
+        return make<int32_t>(std::move(col), SType::INT32, gby, is_grouped);
         case SType::DATE32:
-          return make<int32_t>(std::move(col), gby, is_grouped);
+          return make<int32_t>(std::move(col), SType::DATE32, gby, is_grouped);
         case SType::INT64:
+        return make<int64_t>(std::move(col), SType::INT64, gby, is_grouped);
         case SType::TIME64:
-          return make<int64_t>(std::move(col), gby, is_grouped);
+          return make<int64_t>(std::move(col), SType::TIME64, gby, is_grouped);
         case SType::FLOAT32:
-          return make<float>(std::move(col), gby, is_grouped);
+          return make<float>(std::move(col), SType::FLOAT32, gby, is_grouped);
         case SType::FLOAT64:
-          return make<double>(std::move(col), gby, is_grouped);
+          return make<double>(std::move(col), SType::FLOAT64, gby, is_grouped);
```

Comment:
Passing `stype` is not needed here either.

Reply:
stype passed here is used to cast in place